File: core/src/main/java/io/snabble/sdk/CustomerCardInfo.java
package io.snabble.sdk;

public class CustomerCardInfo {
    private String cardId;
    private boolean isRequired;

    CustomerCardInfo(String cardId, boolean isRequired) {
        this.cardId = cardId;
        this.isRequired = isRequired;
    }

    public String getCardId() {
        return cardId;
    }

    public boolean isRequired() {
        return isRequired;
    }
}

File: core/src/main/java/io/snabble/sdk/CustomerCardInfo.kt
package io.snabble.sdk;

public class CustomerCardInfo {
    private final String cardId;
    private final boolean isRequired;

    CustomerCardInfo(String cardId, boolean isRequired) {
        this.cardId = cardId;
        this.isRequired = isRequired;
    }

    public String getCardId() {
        return cardId;
    }

    public boolean isRequired() {
        return isRequired;
    }
}